package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "react-mentoring",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "webpack serve --config webpack.dev.config.js",
```

Comment:
Note: you can add `--open` flag so that after `npm start` the page will automatically opens in the browser.

`"start": "webpack serve --open <...>"`

But you probably have already known this. Then just resolve this thread )